_getting-started/index.md

```diff
@@ -7,55 +7,117 @@ includeTOC: true
-* [Building a Scala Project with IntelliJ and sbt](/getting-started/intellij-track/building-a-scala-project-with-intellij-and-sbt.html)
-* [Testing Scala in IntelliJ with ScalaTest](/getting-started/intellij-track/testing-scala-in-intellij-with-scalatest.html)
-
-
-### If you prefer working on the command line...
-
-If you prefer using a text editor like emacs, Vim, Atom, or Sublime Text, then
-the best way to compile, test, and run Scala code is by using _sbt_, Scala's build
-tool.
-
-* [Getting Started with Scala and sbt on the Command Line](/getting-started/sbt-track/getting-started-with-scala-and-sbt-on-the-command-line.html)
-* [Testing Scala with sbt and ScalaTest on the Command Line](/getting-started/sbt-track/testing-scala-with-sbt-on-the-command-line.html)
-
-<!-- sbt is the easiest way to ensure that your Scala project is reproducible;
-you specify a Scala version, any libraries you depend on, and sbt takes care of
-the rest, so it's as easy as possible for someone else to compile and run your
-Scala project. -->
 
 ## Next Steps
 Once you've finished these tutorials, check out:
 
-* [The Tour of Scala](/tour/tour-of-scala.html) for bite-sized introductions to Scala's features.
 * [The Scala Book](/overviews/scala-book/introduction.html), which provides a set of short lessons introducing Scala’s main features.
+* [The Tour of Scala](/tour/tour-of-scala.html) for bite-sized introductions to Scala's features.
 * [Learning Resources](/learn.html), which includes online interactive tutorials and courses.
 * [Our list of some popular Scala books](/books.html).
 
 ## Getting Help
 There are a multitude of mailing lists and real-time chat channels in case you want to quickly connect with other Scala users. Check out our [community](https://scala-lang.org/community/) page for a list of these resources, and for where to reach out for help.
+
+<!-- Hidden elements whose content are used to provide OS-specific download instructions.
+ -- This is handled in `resources/js/functions.js`.
+ -->
+    
+<div style="display:none" id="stepOne-linux">
+       <code class="hljs">$ curl -Lo cs https://git.io/coursier-cli-linux && chmod +x cs && ./cs setup </code> <br>
+</div>
+ 
+<div style="display:none" id="stepOne-unix">
+    <p>Follow <a href="https://get-coursier.io/docs/cli-overview.html#install-native-launcher" target="_blank">the instructions to install the <code>cs</code> launcher</a> then run:</p>
+    <p><code>$ ./cs setup</code></p>
+</div>
+ 
+<div style="display:none" id="stepOne-osx">
+    <div class="highlight">
+        <code class="hljs">$ curl -Lo cs https://git.io/coursier-cli-macos && chmod +x cs &&  (xattr -d com.apple.quarantine cs || true) && ./cs  setup </code> <br>
+    </div>
+    <p>Alternatively you can use Homebrew:</p>
+        <div class="highlight">
+            <code class="hljs">$ brew install coursier/formulas/coursier && cs setup </code> <br>
+        </div>
```

Comment:
I’m not familiar with homebrew but the instructions look simpler. Should we put them first?

Reply:
Good idea. I kept the same order that we can found in coursier-cli install page